include/mgclient.h

```diff
@@ -1233,7 +1233,7 @@ MGCLIENT_EXPORT const char *mg_session_params_get_username(
     const mg_session_params *);
 MGCLIENT_EXPORT const char *mg_session_params_get_password(
     const mg_session_params *);
-MGCLIENT_EXPORT const char *mg_session_params_get_client_name(
+MGCLIENT_EXPORT const char *mg_session_params_get_user_agent(
```

Comment:
This never worked properly... The `..._client_name` doesn't exist in the implementation file.